coordinator/persistence-dse-next/src/main/java/io/stargate/db/cassandra/impl/Conversion.java

```diff
@@ -566,7 +566,7 @@ public static Column.ColumnType getTypeFromInternal(AbstractType abstractType) {
       TupleType tupleType = ((TupleType) abstractType);
       return Column.Type.Tuple.of(
               ((TupleType) abstractType)
-                  .allTypes().stream()
+                  .subTypes().stream()
```

Comment:
Is this new functionality?

Reply:
I see this has been changed in this commit https://github.com/datastax/cassandra/pull/891/files#diff-2f66334dab1923767bcb02c37ae8009cab617d21c4586c5a34be8803aac0263b in the `vsearch` branch of `datastax/cassandra`

Reply:
Ok but is that just rename or actual change in behavior?